src/styles/modules/button.scss

```diff
@@ -1,12 +1,45 @@
-.button{
-   background-color: #ff2d2d;
-   color: #ffffff;
-   padding: 10px 53px;
-   border-radius: 5px;
-   font-size: 18.5px;
-   font-weight: bold;
-   font-stretch: normal;
-   font-style: normal;
-   line-height: 1.84;
-   letter-spacing: 0.46px;
-}
\ No newline at end of file
+.button-primary {
```

Comment:
look you should have your main class `button` which will have all common styles like paddings border-radius, etc, and your modificatory classes like `button-primary button-secondary` will have styles they need specifically, which differs, like `background color`